api/src/main/java/org/eclipse/microprofile/health/Startup.java

```diff
@@ -45,15 +45,15 @@
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target({TYPE, METHOD, PARAMETER, FIELD})
```

Comment:
I noticed this annotation can be used on class, method, parameter and field. The TCKs only assert on class level annotation. As per my comments on the other PR regarding add this target on Liveness and Readiness, I suggest some tests to be added to ensure the usage on method, parameter, field were honoured. Thoughts?

Reply:
@Emily-Jiang good idea. Can you please raise it as an issue? We shouldn't discuss that in this PR.